schemas/alpha/azure.yaml.json

```diff
@@ -541,16 +600,21 @@
                     "title": "The platform target",
                     "default": "amd64"
                 },
-                "tag": {
-                    "type": "string",
-                    "title": "The tag that will be applied to the built container image.",
-                    "description": "If omitted, a unique tag will be generated based on the format: {appName}/{serviceName}-{environmentName}:azd-deploy-{unix time (seconds)}. Supports environment variable substitution. For example, to generate unique tags for a given release: myapp/myimage:${DOCKER_IMAGE_TAG}"
-                },
```

Comment:
Do you have more context around this change? I think we're making a (potential breaking?) change to make `tag` split into `image:tag`. The old name may be confusing but it was meant to be what you would apply with `docker tag`.

It's harder to evaluate the tradeoff proposed by this feature change within the other bigger feature change

Reply:
This wasn't actually removed, just showing this way in the diff for some reason.  `registry` and `image` are added to the docker config.

Reply:
I don't mean it's removed.

It used to be:

```
tag: myapp:azdeploy-123
```

now it's

```
image: myapp
tag: azdeploy-123
```

which is a behavioral feature change

Reply:
Yes, you are correct.  In this updated model the user can specify an image & tag in the `image` property or specify them separately if needed.

```yaml
image: myapp:azddeploy-123
```

is equivalent to: 

```yaml
image: myapp
tag: azddeploy-123
```